index.html

```diff
@@ -2,10 +2,51 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
-    <link rel="stylesheet" href="css/style.css">
+    <title>On The Other Side</title>
+    <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville" rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/stylesheet.css">
+    <title>On The Other Side by Gabriel Lu</title>
   </head>
   <body>
-    <script src="js/script.js" charset="utf-8"></script>
+    <h1> On The Other Side </h1>
+    <h3> <br> </h3>
```

Comment:
what's the `<br>` for?

Reply:
I had put it to push down the game halves lower from the h1 element